pkgs/development/python-modules/proton-vpn-network-manager-wireguard/default.nix

```diff
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  gobject-introspection,
+  setuptools,
+  proton-core,
+  proton-vpn-killswitch-network-manager-wireguard,
+  proton-vpn-network-manager,
+  nix-update-script,
+}:
+
+buildPythonPackage rec {
+  pname = "proton-vpn-network-manager-wireguard";
+  version = "0.1.0";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ProtonVPN";
+    repo = "python-proton-vpn-network-manager-wireguard";
+    rev = "v${version}";
+    hash = "sha256-ta3PrggudGTpf7bdMk0Djw903Dw6Ji0vXl42bc6QGmk=";
+  };
+
+  nativeBuildInputs = [
+    # Solves Namespace NM not available
+    gobject-introspection
+    setuptools
+  ];
+
+  propagatedBuildInputs = [
```

Comment:
```suggestion
  dependencies = [
```

Reply:
Should this be done to all modules then? Can we do this in a one-commit follow-up please it was already a bunch of work to split this :/

Reply:
in general yes. If a review comment points out a systemic issue it is best to fix all occourances. Most reviewers don't have the bandwith to go thrugh the entire diff and mark each instance.